Examples/Python/ellipsoid.py

```diff
@@ -88,28 +88,28 @@ def Run_Pipeline(args):
     if int(args.interactive) != 0:
         input("Press Enter to continue")
 
-    parentDir = 'TestEllipsoids/PrepOutput/'
+    parentDir = 'TestEllipsoids/groomed/'
```

Comment:
Do we have the wrong zip file?  On a clean system (GitHub Action), this is failing.

@AtefehKashani @0leks

Reply:
I guess this isn't the right place.  But in any case, the automated test doesn't work while looking for some file.

Reply:
Ok, it was the path that the test was looking for output.

Reply:
@akenmorris  Sorry I missed you r massages earlier. Did this solve the problem?

Reply:
I committed the fix.